api/src/main/java/org/openmrs/module/covid19/COVID19Activator.java

```diff
@@ -118,37 +122,46 @@ private void setupEncounters() {
 	
 	private void setupConcepts() {
 		// TODO These will eventually be replaced by CIEL
-		Concept caseDiagnosed = new Concept();
-		caseDiagnosed.setUuid("165903A");
-		
-		ConceptName caseDiagnosedName = new ConceptName();
-		caseDiagnosedName.setLocale(Locale.ENGLISH);
-		caseDiagnosedName.setLocalePreferred(true);
-		caseDiagnosedName.setName("Country where diagnosed");
-		
-		ConceptDatatype coded = conceptService.getConceptDatatypeByName("Coded");
+		//		Concept caseDiagnosed = new Concept();
+		//		caseDiagnosed.setUuid("165903A");
+		//		
+		//		ConceptName caseDiagnosedName = new ConceptName();
+		//		caseDiagnosedName.setLocale(Locale.ENGLISH);
+		//		caseDiagnosedName.setLocalePreferred(true);
+		//		caseDiagnosedName.setName("Country where diagnosed");
+		//		
+		//		ConceptDatatype coded = conceptService.getConceptDatatypeByName("Coded");
+		//		
+		//		caseDiagnosed.addName(caseDiagnosedName);
+		//		caseDiagnosed.setConceptClass(conceptService.getConceptClassByName("Question"));
+		//		caseDiagnosed.setDatatype(coded);
+		//		
+		//		caseDiagnosed.addAnswer(new ConceptAnswer(conceptService.getConceptByMapping("165820", "CIEL")));
+		//		
+		//		conceptService.saveConcept(caseDiagnosed);
+		//		
+		//		Concept concept = conceptService.getConceptByMapping("162689", "CIEL");
+		//		concept.setDatatype(conceptService.getConceptDatatypeByName("Text"));
+		//		conceptService.saveConcept(concept);
+		//		
+		//		for (String code : new String[] { "165198", "165655" }) {
+		//			concept = conceptService.getConceptByMapping(code, "CIEL");
+		//			concept.setDatatype(coded);
+		//			conceptService.saveConcept(concept);
+		//		}
+		//		
+		//		concept = conceptService.getConceptByMapping("165795", "CIEL");
+		//		concept.setConceptClass(conceptService.getConceptClassByName("Finding"));
+		//		concept.setDatatype(coded);
```

Comment:
Instead of commenting this out, just delete it.

Reply:
Sure i deleted this out